.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I wish we could get rid of this file. But I guess it is the only way that will prevent the package-lock.json from being generated?

Reply:
Yes. Or at least the only I found.
The `package-lock.json` is also present in the `.gitignore` so it won't be pushed to the repo.

The `package-lock=false` prevent the `package-lock.json` to be generated locally. So when you do `npm install` on a repo that you cloned a while, you'll get the last dependencies.

If we were to remove it, maintainers/contributors would have to do `rm package-lock.json && rm node_modules && npm install` to get the last dependencies. Probably not a big deal.

With or without it, it's up to the maintainers/contributors to do update their local dependencies regularly. And I'm not sure their an automated way to make sure everyone have updated dependencies...